src/hotspot/share/cds/heapShared.cpp

```diff
@@ -280,10 +280,10 @@ oop HeapShared::archive_object(oop obj) {
     return ao;
   }
 
-  int len = obj->size();
+  int len = (int)obj->size();
```

Comment:
This code is used by `INCLUDE_CDS_JAVA_HEAP` and is unrelated to custom class loaders.